docs/Remote-Workspaces-(v0.25.0+).md

```diff
@@ -191,33 +191,39 @@ Lake. However we can use the bundled `zapi` command line tool to create a Pool
 in the Zed Lake for our data and then use the bundled `brimcap` to load data
 into it.
 
-As sample packet data, we'll import a [wrccdc pcap](https://wrccdc.org/) from a
-separate shell on our Linux VM:
+As sample packet data, we'll import and index a [wrccdc pcap](https://wrccdc.org/)
+from a separate shell on our Linux VM:
 
 ```
 ubuntu# wget --quiet https://archive.wrccdc.org/pcaps/2018/wrccdc.2018-03-23.010014000000000.pcap.gz
 ubuntu# gunzip wrccdc.2018-03-23.010014000000000.pcap.gz
-ubuntu# /opt/Brim/resources/app.asar.unpacked/zdeps/zapi create -p wrccdc
-ubuntu# /opt/Brim/resources/app.asar.unpacked/zdeps/brimcap load -p wrccdc -root ~/.config/Brim/data/brimcap-root wrccdc.2018-03-23.010014000000000.pcap
-100.0% 476MiB858KiB604B/476MiB858KiB604B records=670822
+ubuntu# export ZDEPS="/opt/Brim/resources/app.asar.unpacked/zdeps"
```

Comment:
Maybe set `PATH` instead?

Reply:
@nwt: Yep. Going with the theme, I've gone the `PATH` route instead of `ZDEPS/`. Thanks.